app/src/main/java/com/simplecity/amp_library/ui/screens/nowplaying/PlayerFragment.java

```diff
@@ -236,8 +238,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                 } else {
                     uiEventType = UiEventType.PAUSE;
                 }
-                UiEvent uiEvent = EventUtils.newUiEvent(song, uiEventType, getContext());
-                FirebaseIOUtils.saveUiEvent(uiEvent);
+                newUiEvent(song, uiEventType, getContext(), Long.MAX_VALUE);
```

Comment:
You should call `newUiEvent(song, uiEventType, getContext())` instead of passing in the `Long.MAX_VALUE`.